### DISCUSSION TITLE
Support for model-integrated UV anim

### DISCUSSION THREAD

**Discussion by tut001** [multitheftauto/mtasa-blue#981] [Ideas]
**Is your feature request related to a problem? Please describe.**
Custom models using UV animation does not seem to be featured in MTA. The animation data is stored in the model and does not require IDE or IPL changes.

**Describe the solution you'd like**
I'd like to see the UV anim data be supported for MTA.

**Describe alternatives you've considered**
See [this page](https://wiki.multitheftauto.com/wiki/Shader_examples), it has a resource that lets you achieve the same result as if the UV animation was integrated in the model.

**Additional context**
This has, from my understanding never been possible on MTA. Had a look around threads from 2009-2016.

**Comment by ghost** [comment]
Does it work in SA if you replace the DFF file in gta3.img?

**Comment by tut001** [comment]
Hello @saml1er, thank you very much for the reply.

I have tested it on single player with gta3.img modification and it works fine. On MTA the data is not read. I have recorded a video on SA and MTA respectively:
https://streamable.com/2017f
https://streamable.com/a98fu
Also, here's a copy of the mod. I did not create it.
https://cdn.discordapp.com/attachments/308955445568077824/587002409570664449/uvanim.rar
